src/systems/kinetic_energy_monitor/KineticEnergyMonitor.cc

```diff
@@ -124,6 +132,8 @@ void KineticEnergyMonitor::Configure(const Entity &_entity,
   ignmsg << "KineticEnergyMonitor publishing messages on "
     << "[" << topic << "]" << std::endl;
 
+  this->dataPtr->haltMode = sdfClone->Get<bool>("halt_mode", false).first;
```

Comment:
Can you add documentation about `halt_mode` in `KineticEnergyMonitor.hh`?

Reply:
Added on  f5c472e